gcloud-java-examples/README.md

```diff
@@ -89,6 +89,20 @@ To run examples from your command line:
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="download <bucket_name> test.txt"
     mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.storage.StorageExample" -Dexec.args="delete <bucket_name> test.txt"
     ```
+ * Here's an example run of `Stat`.
+
+     Before running the example, go to the [Google Developers Console][developers-console] to ensure that Google Cloud Storage API is enabled and that you have a bucket with a file in it.
+     Compile the JAR with:
+    ```
+    mvn package -DskipTests -Dmaven.javadoc.skip=true  -Dmaven.source.skip=true
+    ```
+    Then run the sample with:
+    ```
+    java -cp gcloud-java-contrib/gcloud-java-nio/target/gcloud-java-nio-0.1.6-SNAPSHOT-shaded.jar:gcloud-java-examples/target/gcloud-java-examples-0.1.6-SNAPSHOT.jar com.google.gcloud.examples.Stat --check
+    ```
+    The sample doesn't have anything about GCS in it: it gets that ability from the nio jar that
+    we're adding to the classpath. When you do this in practice you also have to add all of its
```

Comment:
Actually I removed it since the nio jar is "fat": it includes its dependencies so you don't need to do anything special. I added a bit about how when we can it's better to just add a dependency in the pom and recompile.